broker/broker-undelivered-messages-ui/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "message-broker",
+  "version": "1.0.0",
+  "description": "Configurations and controls for the message broker.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Would be best to move the `standard ./src/**/*.js` script from `pretest` to `test` and then remove the pretest phase all together since there are only checkstyle tests so far.